<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortedKeys</class>
 <widget class="QDialog" name="SortedKeys">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>353</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Sorted Keys</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../usr/share/machinetrans/resources/logo.png</normaloff>../../../../usr/share/machinetrans/resources/logo.png</iconset>
  </property>
  <widget class="QComboBox" name="key1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="key2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>200</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="key3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>80</y>
     <width>200</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="key5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>200</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="key4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>200</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="key6">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>200</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="key8">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>200</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="key7">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>200</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="key9">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>220</y>
     <width>200</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="quitBttn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptBttn">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>260</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QPushButton" name="helpBttn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>0</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleLbl">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>411</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select Sort Keys</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="key1Lbl">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Key 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="key2Lbl">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Key 2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="key3Lbl">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>50</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Key 3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="key4Lbl">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Key 4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="key5Lbl">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Key 5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="key6Lbl">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>120</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Key 6</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="key7Lbl">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Key 7</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="key8Lbl">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Key 8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="key9Lbl">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>190</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Key 9</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="resetBttn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
